packages/augur-ui/src/modules/trading/components/form.tsx

```diff
@@ -256,6 +278,44 @@ class Form extends Component<FromProps, FormState> {
     return { isOrderValid: passedTest, errors, errorCount };
   }
 
+  findNearestValues = value => {
+    const { market } = this.props;
+    let tradeInterval = DEFAULT_TRADE_INTERVAL;
+    let numTicks = market.numTicks;
+
+    if (!numTicks) {
+      numTicks = tickSizeToNumTickWithDisplayPrices(
+        createBigNumber(market.tickSize),
+        createBigNumber(market.minPrice),
+        createBigNumber(market.maxPrice)
+      );
+    }
+
+    if (market.marketType == SCALAR) {
+      tradeInterval = getTradeInterval(
```

Comment:
Can get interval be used for all market types. Just to make the code simpler and consistent